include/battle.h

```diff
@@ -1137,6 +1130,13 @@ static inline u32 GetBattlerSide(u32 battler)
     return GetBattlerPosition(battler) & BIT_SIDE;
 }
 
+static inline u32 GetMoveType(u32 move)
+{
+    if (gBattleStruct->dynamicMoveType)
```

Comment:
Probably this needs a gMain.inBattle, since the function name makes me think that it could be used outside of battle context.

Reply:
Good catch. it is also used for the summary

Reply:
done. had to move the function to `battle_util.c`.